HouseRules_Essentials/Rulesets/NoSurprisesRuleset.cs

```diff
@@ -0,0 +1,20 @@
+namespace HouseRules.Essentials.Rulesets
+{
+    using HouseRules.Essentials.Rules;
+    using HouseRules.Types;
+
+    internal static class NoSurprisesRuleset
+    {
+        internal static Ruleset Create()
+        {
+            const string name = "No Surprises.";
+            const string description = "This prevents any surprises in the dark or coming through doors.";
```

Comment:
Small nits:
1. Maybe we shouldn't include trailing periods in the name.  For simplicity/consistency.
2. Instead of `This prevents ...`, WDYT about just `No surp...` or `Prevent any surp...`
  - `This prevents...` just sort of reads like if this were a rule rather than a rulset.

Reply:
Agreed and fixed.